.jscsrc

```diff
@@ -1,4 +1,10 @@
 {
   "preset": "airbnb",
-  "excludeFiles": ['dist']
+  "excludeFiles": ['dist'],
+  "requireSpacesInsideObjectBrackets": false,
+  "requireCamelCaseOrUpperCaseIdentifiers": {
+    allExcept: [
+      "child_process",
```

Comment:
What we can do is add a test to make sure these are JSON parseable. Problem solved.